stories/settings.stories.js

```diff
@@ -31,7 +31,9 @@ import { __ } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
-import SettingsModules from '../assets/js/components/settings/SettingsModules';
+// import SettingsModules from '../assets/js/components/settings/SettingsModules';
```

Comment:
Let's remove this line if it is not needed anymore.

Reply:
👍